### ISSUE TITLE
Connection Errored Out on Paperspace

### ISSUE THREAD

**Issue by AIShank** [TheLastBen/fast-stable-diffusion#2153]
Trying to do anything within the UI gives me this error, using the latest notebook/did a fresh install and I'm still getting the error.

![firefox_fE92J82tQj](https://github.com/TheLastBen/fast-stable-diffusion/assets/115922400/9d3cb940-c8ee-4a45-b313-c8dc646e9307)

No errors are showing up in the cell either.
![firefox_tP22XeHHzp](https://github.com/TheLastBen/fast-stable-diffusion/assets/115922400/4df54e18-5ae1-4362-a795-6dbd43ba2710)

**Comment by stlaurentjr** [comment]
Same problem.
This is incredibly frustrating.
Initially, I assumed the issue was related to the ControlNet parameters, but then I discovered that it persists even without using Automatic1111

**Comment by TheLastBen** [comment]
when you click on generate, do you get any error msg ?

**Comment by CoqueTornado** [comment]
I have same errors everywhere, some say 
Error:Something went wrong
Connection errored out.

others appears like this
![arggg](https://github.com/TheLastBen/fast-stable-diffusion/assets/109884184/d5ada0fd-7be0-4ff9-8610-15059a95e6fd)

**Comment by CoqueTornado** [comment]
I can generate txt to image or even deforums but not changing parameters if I select another tab, like 3D, there will appear errors

**Comment by TheLastBen** [comment]
does it display any error msg in the notebook cell

**Comment by AndresTarqui** [comment]
Same thing happens to me, but i tried opening the UI on my cellphone and seems to work as usual, and nothing appears on the cell output.

**Comment by AIShank** [comment]
Something I noticed is that the startup cell isn't generating a gradio link, instead its this tensorbord thing.
![Screenshot_20230511-120007](https://github.com/TheLastBen/fast-stable-diffusion/assets/115922400/9f5b4e09-8643-4897-bd23-a0cc142bf053)

Pretty sure the last time I ran the AI a few days ago it was using a gradio link and I wasn't getting these errors. Not sure if it's related or not but figured I'd mention it.

**Comment by AIShank** [comment]
Is there anything here that seems like an issue? Haven't been getting any errors in the cell log but this is incredibly frustrating to deal with.

**Comment by TheLastBen** [comment]
add --share in the last cell right after $configf like this `$configf --share`

**Comment by TheLastBen** [comment]
it's a gradio temporary issue

**Comment by AIShank** [comment]
Yeah I just checked their website, says things have been down for a bit regarding share links, hopefully this solution works when it's back up.

Thanks for the responses as always Ben.

**Comment by TheLastBen** [comment]
keep me informed about the functionality of the local paperspace server (without --share)

**Comment by CoqueTornado** [comment]
Hi Ben, I tried to delete all and reinstall it, tried the --share parameter but gradio does not work yet. Tried with some different models, still the error. It looks like it comes from the tensorflow link and gradio out.
Tried another parameters like "--no-gradio-queue"  but still the errors. 
I copy paste this just in case it can be appplied for our issue. The link comes from here https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/9074
"
Two days ago, I managed to finally perform the embedding training. Among the things I did, which resulted in the training finally working, was updating Stable Diffusion from commits https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/22bcc7be428c94e9408f589966c2040187245d81 to commits https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/5ab7f213bec2f816f9c5644becb32eb72c8ffb89, updating Torch from version 1.12.1 to version 2.0.0 (for which the --reinstall-torch flag added after export COMMAND_ARGS= in the webui-user.sh file did not work, and instead I had to delete the venv folder within the Stable-Diffusion-webui main folder first, and then start the Stable Diffusion server as normal, during which startup the latest 2.0.0 version of Torch was automatically downloaded and installed), as well as installing all the libraries for PyTorch, which were indicated as missing, through Home Brew, which were: ninja, eigen, libuv, numpy, openblas, pybind11, python-typing-extensions, pyyaml, and libomp (although I am not confident whether installing those libraries indeed had an impact of making the embedding training finally working).
I trained my embedding with 487 images, which I considered as not being relatively many in order to later generate good quality, realistically-looking images, although way more than 10 images, which is commonly suggested as being enough. Unfortunately, though, the resulting images generated are far from looking realistic. Instead, as you can see in the screenshots I have attached, they are brightly and vividly colored, looking somewhat cartoonish, although the unsharp blending of the colors, which are not unific and definitely bordered, makes them not even resemble typical cartoon drawings. Those images are somewhere in-between photos and cartoon drawings, being not either. I’m not sure whether or not this is due to the latest 2.0.0 version of Torch (which users are warned about causing issues with training, although running version 1.12.1, I couldn’t perform any training at all), the https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/5ab7f213bec2f816f9c5644becb32eb72c8ffb89 commit of Stable Diffusion itself, or something else that Stable Diffusion needs to operate? Is this a known issue currently occurring, and should I simply wait for the future updates of Stable Diffusion?
"
just my 2 cents. Thank you Ben for all!

**Comment by TheLastBen** [comment]
@CoqueTornado thanks, I removed the gradio queue, hopefully it solves the issue

**Comment by CoqueTornado** [comment]
yeah, it works now flawessly! there is hope! the world is beautiful! listen to my pop music in the meanwhile hahah! cheers! and many many many thanks!!!

**Comment by roadev** [comment]
I'm still facing the same errors even using the `--share` flag

**Comment by TheLastBen** [comment]
@roadev how to reproduce the issue ?

**Comment by AIShank** [comment]
Gradio share links seem to be back up, but I'm still getting the same 'no interface is runnig' screen when using the --share command.

**Comment by roadev** [comment]
I started the machine on Paperspace (I'm using the pro tier)
I just wanted to use the automatic1111 so I did:
1. Run install dependencies
2. Download model (v1.5)
3. Run the Test model block (using as source model just one of the models I was previously using)
4. Everything appears as the first posted image of this issue and using the `--share` gives me a gradio link that points to a "No interface is running right now"

**Comment by roadev** [comment]
> @roadev how to reproduce the issue ?

I started the machine on Paperspace (I'm using the pro tier)
I just wanted to use the automatic1111 so I did:

    Run install dependencies
    Download model (v1.5)
    Run the Test model block (using as source model just one of the models I was previously using)
    Everything appears as the first posted image of this issue and using the --share gives me a gradio link that points to a "No interface is running right now"

**Comment by TheLastBen** [comment]
is this the A1111 notebook or dreambooth ?

**Comment by AIShank** [comment]
For me it's the A111 notebook.

**Comment by roadev** [comment]
Dreambooth v1.5 Paperspace Notebook

**Comment by TheLastBen** [comment]
I'm trying to reproduce the error but couldn't, do not use the `--share` flag

**Comment by roadev** [comment]
![Screenshot from 2023-05-16 13-46-11](https://github.com/TheLastBen/fast-stable-diffusion/assets/4525107/7fef21a1-2e15-4784-98db-6856b2b6ff70)

for me it is the same without `--share`

I'm using the tensorboard link generated

**Comment by roadev** [comment]
Got the same result using the A1111 notebook.

**Comment by TheLastBen** [comment]
does it happen immediately after starting the webui ?

**Comment by Jowe79** [comment]
Yesterday me too..same error get errored out....
It's frustrating... it's a ngrok error? Or code itself from notebook? Thanks

**Comment by TheLastBen** [comment]
@Jowe79 I removed ngrok from the notebook, are you using the latest one ?

**Comment by Jowe79** [comment]
I don't know please can you share me the LAST notebook??thanks

**Comment by Jowe79** [comment]
> @Jowe79 I removed ngrok from the notebook, are you using the latest one ?

I dont use paperspace, only google colab pro and your notebook fast stable diffusion the last ben...but what is the last version upfated without ngrok?

**Comment by roadev** [comment]
> @Jowe79 I removed ngrok from the notebook, are you using the latest one ?

I just copied the last code from your notebooks and it runs again, thank you!

**Comment by TheLastBen** [comment]
@Jowe79 you will always find the latest notebooks in the folder named "latest_notebooks", they are updated each time the first cell is ran

**Comment by Jowe79** [comment]
The latest notebooks but where?? Can you give me the link?

**Comment by TheLastBen** [comment]
in the file explorer with the all the folders and files

**Comment by Jowe79** [comment]
Sorry but i using colab notebook...in the section on the left where there are the drive folder etcetc?? Any screenshot for visual? Thanks

--Jordi--

________________________________
De: Ben ***@***.***>
Enviado: miércoles, 24 de mayo de 2023, 22:50
Para: TheLastBen/fast-stable-diffusion ***@***.***>
CC: Jordi ***@***.***>; Mention ***@***.***>
Asunto: Re: [TheLastBen/fast-stable-diffusion] Connection Errored Out on Paperspace (Issue #2153)


in the file explorer with the all the folders and files

**Comment by TheLastBen** [comment]
that doesn't apply to colab, for colab you just use the official link : https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

**Comment by Jowe79** [comment]
But  this colab have maintenance? You'll update for fix bugs etc or get errored out desconnections?

--Jordi--
________________________________
